Library/Formula/rust.rb

```diff
@@ -23,6 +46,36 @@ def install
     system "./configure", *args
     system "make"
     system "make install"
+
+    resource("cargo").stage do
+      cargo_stage_path = pwd
+
+      resource("rustc-nightly-2015-03-30").stage do
+        system "./install.sh", "--prefix=#{cargo_stage_path}/rustc"
+      end if build.stable? && build.without?("brewed-rust")
+
+      resource("cargo-nightly-2015-03-26").stage do
+        system "./install.sh", "--prefix=#{cargo_stage_path}/target/snapshot/cargo"
+        # satisfy make target to skip download
+        touch ("#{cargo_stage_path}/target/snapshot/cargo/bin/cargo")
+      end if build.stable?
```

Comment:
Flip these around.

```ruby
if build.stable?
  resource("cargo-nightly-2015-03-26").stage do
    system "./install.sh", "--prefix=#{cargo_stage_path}/target/snapshot/cargo"
    touch ("#{cargo_stage_path}/target/snapshot/cargo/bin/cargo")
  end
end
```

The ` end if ` statements have been dying for a while.